operations/reservations.md

```diff
@@ -1317,3 +1319,14 @@ Adds a new product order of the specified product to the reservation.
     ]
 }
 ```
+
+#### Commander origin
```

Comment:
suggestion: I think we should explain here the meaning of commander origin and introduce the list as incomplete example or mention that more values may be added in the future, so the developers don't attempt to do strict validation on the value.

```suggestion
#### Commander origin

In case the reservation is created with Mews Operations, `CommanderOrigin` specifies how the reservation was created, for example:
```
```

Reply:
Since this is just definition of enum, you should not put explanation here since you don't know where it could be linked in the future. Explenation is done on level of endpoint 👍

And that enum can change in the future is done via `...` in the end